docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,104 @@
+
+Guardrails for public code are available to all Sourcegraph Enterprise instances and are **disabled** by default. You can enable it from the **Site configuration** or the IDE extension settings. It only matches code snippets that are at least **10 lines** or longer, and it spans **290,000** search results.
+
+### Admin Controls
+
+<Callout type="note">Admin Controls is supported with VS Code IDE extension.</Callout>
+
+Sourcegraph account admins have selective control over users' access to Cody Enterprise, which is now managed via the Sourcegraph role-based access control system. This provides a more intuitive user interface for assigning permission to use Cody.
+
+Using the access controls, you can also configure which Cody capabilities users can have access to (for example, chat, completion, or commands).
+
+### Analytics
+
+<Callout type= "note">Admin Controls is supported with VS Code IDE extension and on the latest versions of JetBrains IDE.</Callout>
+
+Cody Enterprise users get a clear view of usage analytics for their instance on a self-service basis. A separately managed cloud service for Cody analytics (for example, `cody-analytics.sourcegraph.com`) handles user auth, gets metrics data from Sourcegraph's BigQuery instance, and visualizes the metrics data.
+
+The following metrics are available for Cody Enterprise users:
+
+| **Metric Type** |                                                                                                                                                                         **What is measured?**                                                                                                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Active users    | - Total active users <br /> - Average daily users <br /> - Average no. of days each user used Cody (of last 30 days) <br /> - Cody users by day (last 30 days) <br /> - Cody users by month (last two months) <br /> - Cody users by number of days used                                                                                                               |
+| Completions     | - Total accepted completions <br /> - Minutes saved per completion (hardcoded on our end) <br /> - Hours saved by completions (computed based on the previous two) <br /> - Cody completions by day <br /> - Completions acceptance rate <br /> - Weighted completions acceptance rate <br /> - Average completion latency <br /> - Acceptance rate by language <br /> |
```

Comment:
I think we can remove "(hardcoded on our end)" here and after. It was mentioned for additional context on the internal meetings.